abipy/eph/sigeph.py

```diff
@@ -932,6 +1023,16 @@ def interpolate(self, itemp_list=None, lpratio=5, ks_ebands_kpath=None, ks_eband
         #    for spin in range(self.nsppol):
         #        qpdata[spin, ik_ibz, :] = egw_rarr[spin, ik_ibz, :]
 
+        #HM: This seems to do the trick
```

Comment:
Could you add a comment explaining what is the problem you are trying to solve with this portion of code?